scripts/hab_conf_transfer_to_Q.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Transfer `habitat_confirmations.xls` data to `form_fiss_2023.gpkg`"
+date: "Created: 2024-04-26 | Updated: `r format(Sys.Date(), '%Y-%m-%d')`"
+output: 
+  html_document:
+    code_folding: "none"
+params:
+  repo_owner: "NewGraphEnvironment"
+  repo_name: "fish_passage_skeena_2023_reporting"
+---
+
+```{r setup, echo=TRUE, include = TRUE}
+knitr::opts_chunk$set(echo=TRUE, message=FALSE, warning=FALSE, dpi=60, out.width = "100%", eval = FALSE)
+options(scipen=999)
+options(knitr.kable.NA = '--') 
+options(knitr.kable.NAN = '--')
+```
+
+
+## Purpose
+
+This workflow is unique to the 2023 fiss form data and only needs to be done once. While populating the `habitat_confirmations.xls` spreadsheet, edits were made by hand to the `temperature`, `conductivity`, and `turbidity` columns in the `step_2_fish_coll_data` sheet. While doing so we realized we should QA `form_fiss_2023` in Q before populating the `habitat_confirmations.xls` spreadsheet so that we shouldn't need to make edits by hand, like in this case. 
+
+But we learn from our mistakes and will now go and QA `form_fiss_2023`. We don't need to duplicate our edits, so we will copy the edits made by hand in `habitat_confirmations.xls` to `form_fiss_2023.gpkg` before starting the QA. The purpose of this script is to add data from the `habitat_confirmations.xls` spreadsheet to the `form_fiss_2023.gpkg`. `eval = FALSE` because we only need to run this code once, and don't want to run this chunk when knitting the document.
+
+Moving forward we should use the following workflow:
```

Comment:
this is worth more consideration.  i don't think we need to use the main directory too much. nothing to change if the names are correct for now but keep it in mind that we can use the subdirectories, name things so they are ordered naturally and document well so we do as little jumping back and forth as possible.

Reply:
Not following this sorry